tests/functional/services/policy_engine/test_query_by_vulnerability.py

```diff
@@ -0,0 +1,153 @@
+import logging
+from dataclasses import asdict
+from itertools import combinations
+
+import pytest
+
+import tests.functional.services.policy_engine.utils.api as policy_engine_api
+from tests.functional.services.policy_engine.utils.utils import (
+    ImagesByVulnerabilityQuery,
+    ImagesByVulnerabilityQueryOptions,
+)
+
+
+class TestQueryByVulnerability:
```

Comment:
It will be helpful to future debuggers if you include some comments in the bodies of the tests in this class (and the others in this PR that don't have them) to describe the test execution.

Reply:
fixed